.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-  "deepscan.enable": true
+  "deepscan.enable": true,
+  "liveServer.settings.port": 1987,
```

Comment:
Good that you change the port for live server functionality. That improves security. I will only recommend that you add this settings to the .gitignore file in order to do not show them in your public repository.